src/runtime/hexagon_remote/remote.cpp

```diff
@@ -21,6 +21,57 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
+#define NUM_WORKER_THREADS_TO_CREATE 3
+#define STACK_SIZE 4096
```

Comment:
It is quite flexible. There isn't a good reason for it to be 4 kb other than the fact that in a different implementation of the thread pool it had been 4 kb and I just overlooked changing it. What do  you think I should start off with?